examples/bootstrap.php

```diff
@@ -3,7 +3,7 @@
 /**
  * Bootstrap the library
  */
-require_once __DIR__ . '/../vendor/autoload.php';
+require_once __DIR__ . '/../src/OAuth/bootstrap.php';
```

Comment:
Vendor folder doesn't exist, so none of these examples are working because of that. Is there something that I miss?